spec/controllers/account/admin/billing_contracts_imports_controller_spec.rb

```diff
@@ -0,0 +1,115 @@
+require 'rails_helper'
+
+RSpec.describe Account::Admin::BillingContractsImportsController, type: :controller do
+  render_views
```

Comment:
you don't need to login user?

Reply:
> you don't need to login user?

there is login_user on 13 line, or you mean it needed above render_views?